scripts/task.ts

```diff
@@ -278,7 +279,10 @@ async function runTask(task: Task, details: TemplateDetails, optionValues: Passe
 
     return controller;
   } catch (err) {
-    if (junitFilename) await writeJunitXml(getTaskKey(task), details.key, startTime, err);
+    // If there's a non-test related error (junit report has not been reported already), we report the general failure in a junit report
+    if (junitFilename && !existsSync(junitFilename)) {
```

Comment:
Please don't use sync `fs` method? Looks like this is a async function already, `pathExists` is an async equivalent

Reply:
updated!